client/src/test/extension.test.ts

```diff
@@ -1,69 +1,61 @@
 // import * as myExtension from '../extension';
 
-// Defines a Mocha test suite to group tests of similar kind together
-suite('Extension Tests', () => {
-    test('YML extension exists.', () => {
-        assert.strictEqual(
-            true,
+// Defines a Mocha it suite to group its of similar kind together
+describe('Extension Tests', () => {
+    it('YML extension exists.', () => {
+        expect(
             vscode.extensions.all.some((extension) => {
                 return extension.id === YML_EXTENSION_ID;
             }),
-        );
+        ).toBeTruthy();
     });
 
-    test("not existing extension doesn't exists.", () => {
-        assert.strictEqual(
+    it("not existing extension doesn't exists.", () => {
+        expect(
             vscode.extensions.all.some((extension) => {
                 return extension.id === 'Yseop.notExistingExtension';
             }),
-            false,
-        );
+        ).toBeFalsy();
     });
 
-    test('not activated YML extension is not active.', () => {
+    it('not activated YML extension is not active.', () => {
         const ymlExtension = vscode.extensions.all.find((extension) => {
             return extension.id === YML_EXTENSION_ID;
         });
-        assert.notStrictEqual(ymlExtension, null);
-        assert.strictEqual(ymlExtension.isActive, false);
+        expect(ymlExtension).not.toBeNull();
+        expect(ymlExtension.isActive).toBeFalsy();
     });
 
-    test('YML extension activation activates it.', (done) => {
+    it('YML extension activation activates it.', (done) => {
         const ymlExtension = vscode.extensions.all.find((extension) => {
             return extension.id === YML_EXTENSION_ID;
         });
-        assert.notStrictEqual(ymlExtension, null);
-        assert.strictEqual(ymlExtension.isActive, false);
+        expect(ymlExtension).not.toBeNull();
+        expect(ymlExtension.isActive).toBeFalsy();
         ymlExtension
             .activate()
             .then(() => {
-                assert.strictEqual(ymlExtension.isActive, true);
+                expect(ymlExtension.isActive).toBeTruthy();
             })
             .then(done, done);
     });
 
-    test('YML extension adds YML support.', (done) => {
+    it('YML extension adds YML support.', (done) => {
         const ymlExtension = vscode.extensions.all.find((extension) => {
             return extension.id === YML_EXTENSION_ID;
         });
-        assert.notStrictEqual(ymlExtension, null, "Extension doesn't exist.");
+        expect(ymlExtension).not.toBeNull();
         vscode.languages
             .getLanguages()
             .then((languages) => {
-                assert.notStrictEqual(languages, null);
-                assert.notStrictEqual(languages.length, 0);
-                assert.notStrictEqual(languages.indexOf('yml'), -1);
+                expect(languages).not.toBeNull();
+                expect(languages.length).not.toBe(0);
```

Comment:
Nothing like `expect(x).toBeEmpty()`? Just asking xD

Reply:
It seems it [doesn't exist](https://jestjs.io/docs/en/using-matchers). Maybe I can use `is([])` instead?

Reply:
Nonono, it’s OK.